src/ntimporters/utils.py

```diff
@@ -238,3 +236,31 @@ def post_tag(nt_client, tag_name: str, color: str):
         )
     )
     return str(nt_tag.id) if nt_tag else None
+
+
+def match_nt_users(nt_client, emails: list) -> dict:
+    """Match 3rd party with Nozbe users and return email,member id pairs"""
+
+    def md5(email: str, user_id: str):
+        "Calculate hash"
+        return hashlib.md5((user_id + email.lower()).encode(encoding="utf-8")).hexdigest()  # nosec
+
+    nt_users = [
+        (str(elt.email), str(elt.id))
+        for elt in apis.UsersApi(nt_client).get_users()
+        if hasattr(elt, "email")
```

Comment:
placeholder users may have email stored in `invitation_email` field, would be good to match them as well